tiledb/sm/filter/filter_buffer.h

```diff
@@ -339,7 +339,7 @@ class FilterBuffer {
     shared_ptr<Buffer> underlying_buffer_;
 
     /** True if this instance is a view on the underlying buffer. */
-    bool is_view_{false};
+    bool is_view_;
```

Comment:
`is_view_` (and `FilterPipeline::max_chunk_size_`) are always initialized by the constructors, so this is not necessary. And if they ever don't, the compiler will warn-as-error again.